SWEBackend/SWEBackend/Migrations/20210325204723_Initial.cs

```diff
@@ -0,0 +1,175 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Metadata;
+
+namespace SWEBackend.Migrations
+{
+    public partial class Initial : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: "SWEFile",
+                columns: table => new
+                {
+                    Id = table.Column<int>(type: "integer", nullable: false)
+                        .Annotation("Npgsql:ValueGenerationStrategy", NpgsqlValueGenerationStrategy.IdentityByDefaultColumn),
+                    ByteData = table.Column<byte[]>(type: "bytea", nullable: true)
+                },
+                constraints: table =>
+                {
+                    table.PrimaryKey("PK_SWEFile", x => x.Id);
+                });
+
+            migrationBuilder.CreateTable(
+                name: "TimeSlots",
```

Comment:
Sorry for not communicating this sooner, he wanted us to change the time slots to also include a duration field. Would we be able to add the logic here for it.

Reply:
No worries, I must have missed that conversation. Cant that be calculated from the start and end time though?

Reply:
Yes it can, wasn't sure where we wanted to perform that calculation.

Reply:
Probably client side but we can add it to the api endpoint on the server if needed down the line easily enough